worlds/pokemon_rb/__init__.py

```diff
@@ -520,6 +520,12 @@ def pre_fill(self) -> None:
                     else:
                         raise Exception("Failed to remove corresponding item while deleting unreachable Dexsanity location")
 
+    def get_pre_fill_items(self) -> typing.List["Item"]:
+        pool = [self.create_item(mon) for mon in poke_data.pokemon_data]
```

Comment:
What is this about?

Reply:
If `get_all_state` is called before pre_fill, evolution locations will not be accessible. So for the sake of plando, we assume every Pokemon is available.

Reply:
Note that the additional argument added to `get_all_state` allows for the actual pre_fill usage to be unchanged.